lib/cooked_post_processor.rb

```diff
@@ -60,16 +60,8 @@ def grant_badges
   def keep_reverse_index_up_to_date
     upload_ids = Set.new
 
-    @doc.css("a[href]").each do |a|
-      href = a["href"].to_s
-      if upload = Upload.get_from_url(href)
-        upload_ids << upload.id
-      end
-    end
-
-    @doc.css("img[src]").each do |img|
-      src = img["src"].to_s
-      if upload = Upload.get_from_url(src)
+    @doc.css("a/@href", "img/@src").each do |media|
```

Comment:
Wouldn't this, in combination with the other change, make it so we're checking URLs on domains entirely unrelated to the forum?